pkgs/mariadb-galera/default.nix

```diff
@@ -0,0 +1,48 @@
+{ stdenv, fetchurl, cmake, bison, ncurses, openssl, readline, zlib, perl }:
+
+stdenv.mkDerivation rec {
+  name = "mariadb-galera-${version}";
+  version = "10.0.21";
```

Comment:
there was a concern https://github.com/zalora/microgram/blob/master/pkgs/mariadb/default.nix#L6
but that's mitigated by a pgrp hack, so it's not really a concern anymore